src/test/groovy/com/palantir/gradle/versions/VersionsLockPluginIntegrationSpec.groovy

```diff
@@ -356,6 +279,17 @@ class VersionsLockPluginIntegrationSpec extends IntegrationSpec {
         gradleVersionNumber << GRADLE_VERSIONS
     }
 
+    @Override
+    File addSubproject(String name) {
```

Comment:
What we really need to do is to be able to create a subproject from a nested directory, but that's not currently possible from what I could see. This is the next best thing i.e. given `foo:a` create projects `foo` and `foo:a` and then return the latter.